packages/typescript/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../../tsconfig.base.json",
-  "include": ["src/**/*", "types/**/*"]
+  "include": ["src/**/*", "types/**/*"],
+  "compilerOptions": {
+    "skipLibCheck": false
```

Comment:
I think this is the default, so you shouldn't really need to change it here

https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
You’re correct, it is the default for the compiler. But the project `tsconfig` [sets it to `true`](https://github.com/rollup/plugins/blob/a1746415fd032436f23f134a09b1b9cfabf7ceff/tsconfig.base.json#L12). I tried to be unintrusive and not change it for the whole project.

Reply:
Ah, I missed that. Though it looks like we can remove it in the base config without any additional changes, so that might be the better way to go: https://github.com/rollup/plugins/pull/700

Reply:
If needed, could we roll back this change in order to merge the change for the type that is breaking in https://github.com/rollup/plugins/issues/626 ?

Reply:
I think the thing to do would be to merge https://github.com/rollup/plugins/pull/700 instead of changing this file, but perhaps @shellscape could weigh in on whether we can merge that one

Reply:
#700 is merged, please @geigerzaehler can you revert this change?